cmd/contour/serve.go

```diff
@@ -173,8 +175,13 @@ type serveContext struct {
 	httpsPort      int
 	httpsAccessLog string
 
+	// serveInsecure d `json:"serveinsecure"`ver gRPC for serving.
```

Comment:
I think your editor ate this line, I'd prefer to call it something like this
```
// DisableCertificateAuth disables mTLS between Envoy and Contour.
DisableCertificateAuth bool `json:"-"`
```
Let's not make this configurable by the config file -- disable json --users must pass `--insecure` in every invocation of `contour serve`.